packages/manager/modules/pci/src/projects/creating/routing.js

```diff
@@ -17,10 +17,21 @@ export default /* @ngInject */ ($stateProvider) => {
 
       pciProjectsHref: /* @ngInject */ ($state) => $state.href('pci.projects'),
 
-      onProjectDelivered: /* @ngInject */ ($state) => (projectId) =>
-        $state.go('pci.projects.project', {
+      onProjectDelivered: /* @ngInject */ ($state, atInternet, voucherCode) => (
+        projectId,
+      ) => {
+        const trackingData = {
+          name: 'public-cloud::pci::projects::created',
           projectId,
-        }),
+        };
```

Comment:
**suggestion**: Another way to write it that avoids a `if` could be : 
```suggestion
(...voucherCode ? { voucherCode} : {})
        };
```

Reply:
This is done :)